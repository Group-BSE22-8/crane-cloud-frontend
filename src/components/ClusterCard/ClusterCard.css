.ClusterCard {
  width: 15rem;
}

.ClusterCard:hover {
  transition: 0.5s;
  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
  transform: scale(0.95);
}

.CardImageDiv {
  margin: 0;
  width: 100%;
  height: 150px;
  background-color: #008ac1;
  background-repeat: no-repeat;
  background-position: center;
  background-size: contain;
}

.BottomContainer {
  padding: 10px;
  height: 80px;
  display: block;
}

.ClusterCardName {
  width: 100%;
  font-size: 24px;
  padding-top: 5px;
}

.ClusterCardDesc {
  width: 100%;
  font-size: 16px;
  color: #000000;
  padding-top: 10px;
}