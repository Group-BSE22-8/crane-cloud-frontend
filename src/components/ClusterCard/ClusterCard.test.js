import React from 'react';
import { shallow } from "enzyme";
import ClusterCard from './index';

describe('Test the cluster card component', () => {
    const ClusterCardComponent = shallow(<ClusterCard description="" icon="" name="" />);
    it('checks if the cluster card component matches the snapshot', () => {
        expect(ClusterCardComponent).toMatchSnapshot();
    });
});