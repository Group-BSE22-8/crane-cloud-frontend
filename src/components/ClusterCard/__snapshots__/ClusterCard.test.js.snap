// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the cluster card component checks if the cluster card component matches the snapshot 1`] = `ShallowWrapper {}`;
